WcaOnRails/app/views/delegate_reports/_delegate_report.html.erb

```diff
@@ -38,6 +38,12 @@
     <% end %>
   </div>
 
+  <div>
+    <strong>Events</strong>
+    (<%= competition.events.count %>)
+    <%= competition.events.map { |event| event.name }.join(", ") %>
```

Comment:
Just noticed this was on staging so I thought I'd checkout the diff to see if it can be applied easily on top of my work on adding/editing results; in ruby you can easily create a proc returning the attribute using `&:attribute`, ie you can do this:
```suggestion
    <%= competition.events.map(&:name).join(", ") %>
```
Not saying you should change it, just letting you know ;)